src/components/va-alert/va-alert.tsx

```diff
@@ -0,0 +1,117 @@
+import { Component, Event, EventEmitter, Host, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'va-alert',
+  styleUrl: 'va-alert.css',
+  shadow: true,
+})
+export class VaAlert {
+  /* Determines the icon and border/background color.
+   * One of `info`, `error`, `success`, `warning`, or `continue`
+   */
+  @Prop() status: string = 'info';
```

Comment:
Huh - what to do in the absence of a `PropTypes.oneOf` equivalent?  Validation via `@Watch()` decorator?

Reply:
`Watch` is only on updates - it doesn't trigger on an initial render.

Reply:
Given what we have, I am in favor of not trying to be clever here. The one thing I might like to do is add an export somewhere in `vets-website` like in `platform/constants` that contains all of the valid statuses so that it can be imported and used in React apps.

Here are some options we could go with regarding an invalid `status`:

1. Do nothing - if someone passes in an incorrect `status` prop, that isn't the responsibility of the DS team to fix
1. If the `status` is anything other than the pre-defined accepted ones, force it back into the default `info`
1. Leave `status` alone, but send out an error message to let a developer looking at the console know that something is wrong
1. Completely stop the render and throw an error

Option 2 could create annoying debug scenarios (imagine sending in `succes` instead of `success` and then wondering why you are seeing the styling for the `info` status). Option 4 would be pretty bad if it happened in production. Option 3 is pretty close to option 1, but I'm not convinced that it would have much of an impact, especially if the `status` is dynamically generated. I often see errors/warnings get ignored by developers.

Additionally, here is what the styling looks like if `status` is something unexpected. The black border and lack of an icon clearly doesn't match the documented component, so a developer would know that something is wrong.

![image](https://user-images.githubusercontent.com/2008881/115615687-212f0d00-a2a4-11eb-9eb2-c9c93ace35a6.png)